src/search.cpp

```diff
@@ -1250,6 +1250,11 @@ namespace {
       {
           Depth r = reduction<PvNode>(improving, depth, moveCount);
 
+#ifdef ANTI
+          if (pos.is_anti() && pos.can_capture())
+              r -= r ? ONE_PLY : DEPTH_ZERO;
+          else
```

Comment:
I'll likely accept this PR - I am concerned about the possibility of future giveaway patches before concerns about functionality are addressed.

I'll submit anti_lmr7 versus anti_lmr5 at LTC although I'm still struggling to understand how in general LMR should function in giveaway chess.  I suppose I can always attempt to simplify and/or genericize anti_lmr7 later.